server/odc-core/src/main/java/com/oceanbase/odc/core/datasource/SingleConnectionDataSource.java

```diff
@@ -130,6 +130,7 @@ protected void prepareConnection(Connection con) throws SQLException {
 
     private boolean tryLock() {
         try {
+            log.info("get connection try lock, hashcode=" + this.lock.hashCode());
```

Comment:
we'd better get tryLock's result, if we obtain the lock and then we print this lock log~

Reply:
That makes a lot of sense~